pixi.toml

```diff
@@ -0,0 +1,79 @@
+[project]
+authors = ["IRIS-HEP"]
+channels = ["conda-forge"]
+description = "Environments for the Analysis Grand Challenge analyses"
+name = "analysis-grand-challenge"
+platforms = ["linux-64", "osx-64", "osx-arm64"]
+version = "1.4.0"
+
+[tasks]
+
+[dependencies]
+python = ">=3.7"
+pixi-kernel = ">=0.5.1"
+ipykernel = ">=6.29.5"
+pre-commit = ">=4.0.1"
+jupytext = ">=1.16.4"
+ipywidgets = ">=8.1.5"
+pip = ">=24.3.1"
+uv = ">=0.4.27"
+
+[feature.latest.dependencies]
```

Comment:
The "latest" feature assumes no restrictions from run facility and has no upper bounds. I think @alexander-held had suggested that having something like this would be useful.

Reply:
Thanks, yes I believe this is convenient to have.